css_toc_work.py

```diff
@@ -0,0 +1,156 @@
+#
+# Business-logic of "CSS Table of Contents" plugin for ST3
+#
+
+import re
+import heapq
+
+def work_get_section(contents, selected, line):
```

Comment:
Can I rename the definitions? work_ prefix doesn't sound descriptive enough. Or just toc_ prefix to match the plugin?